index.html

```diff
@@ -26,6 +26,20 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+
+      <div id="mobile-nav" class="invisible">
+        <div>
+          <i id="close" class="fas fa-xmark"></i>
+          <div>
+            <li><a class="mobile-link" href="#projects">Portoflio</a></li>
+            <li><a class="mobile-link" href="#about-m">About</a></li>
+            <li>
+              <a class="mobile" href="#contact">Contact</a>
```

Comment:
- [ ] I think if you could fix the _contact_ link by directing to your _contact me_ section when clicked could improve your portfolio's UX. Currently, it takes some time until it redirects to the correct path.